examples/cifar10/alexnet.cc

```diff
@@ -22,16 +22,19 @@
 #include "./cifar10.h"
 #include "singa/model/feed_forward_net.h"
 #include "singa/model/optimizer.h"
+#include "singa/model/updater.h"
 #include "singa/model/initializer.h"
 #include "singa/model/metric.h"
 #include "singa/utils/channel.h"
```

Comment:
pls create another file e.g., multi_gpu.cc, and keep the original single gpu training file.